CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # The Revision History
 
+## v0.2.1 - 2017/09/12
```

Comment:
s/v0.2.1/v0.3.0/

Reply:
and, should change the p-r title.

Reply:
sorry 🙏 !

fixed at https://github.com/bitjourney/react-simple-image/pull/24/commits/4eaa7f932a1dac4ac3eb2828ddd80d2176978e6c